sdk/identity/Azure.Identity/tests/VisualStudioCodeCredentialLiveTests.cs

```diff
@@ -35,379 +30,110 @@ public VisualStudioCodeCredentialLiveTests(bool isAsync) : base(isAsync)
         [RunOnlyOnPlatforms(Windows = true, OSX = true)] // Comment this attribute to run this tests on Linux with Libsecret enabled
         public async Task AuthenticateWithVscCredential()
         {
-            var tenantId = GetTenantId();
             var cloudName = Guid.NewGuid().ToString();
 
-            var fileSystem = CreateTestFileSystemService(cloudName: cloudName);
-            using IDisposable fixture = await CreateRefreshTokenFixtureAsync(tenantId, cloudName);
+            var fileSystem = CredentialTestHelpers.CreateFileSystemForVisualStudioCode(TestEnvironment, cloudName);
+            using IDisposable fixture = await CredentialTestHelpers.CreateRefreshTokenFixtureAsync(TestEnvironment, Mode, ExpectedServiceName, cloudName);
 
-            var options = Recording.InstrumentClientOptions(new VisualStudioCodeCredentialOptions { TenantId = tenantId });
+            var options = Recording.InstrumentClientOptions(new VisualStudioCodeCredentialOptions { TenantId = TestEnvironment.TestTenantId });
             VisualStudioCodeCredential credential = InstrumentClient(new VisualStudioCodeCredential(options, default, default, fileSystem, default));
-            AccessToken token = await credential.GetTokenAsync(new TokenRequestContext(new[] {".default"}), CancellationToken.None);
+            AccessToken token = await credential.GetTokenAsync(new TokenRequestContext(new[] {"https://vault.azure.net/.default"}), CancellationToken.None);
```

Comment:
Session records are updated for `VisualStudioCodeCredentialLiveTests` because of changed scope.